src/modules/account/components/account-deposit/account-deposit.jsx

```diff
@@ -141,9 +61,11 @@ export default class AccountDeposit extends Component {
         </div>
         <div className={Styles.AccountDeposit__main}>
           <div className={Styles.AccountDeposit__description}>
-            {shapeShiftConverter}
-            <div />
-            {airSwapConverter}
+            <div className={Styles.AccountDeposit__0xInstantButton}>
+              <button onClick={openZeroExInstant}>
+                Buy REP using 0x instant.
```

Comment:
Just making sure "instant" is capitalized. Is that done via CSS?

Reply:
Yes.